go/libkb/interfaces.go

```diff
@@ -441,3 +442,50 @@ type GregorListener interface {
 	PushHandler(handler GregorInBandMessageHandler)
 	PushFirehoseHandler(handler GregorFirehoseHandler)
 }
+
+type GlobalContextLite interface {
```

Comment:
APIer, APIerLogger both sound bizarre.

Nothing specific to Proofs about the Context...